x-pack/test_serverless/api_integration/test_suites/observability/cases/find_cases.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import {
+  findCases,
+  createCase,
+  deleteAllCaseItems,
+  postCaseReq,
+  findCasesResp,
+} from './helpers/api';
+
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const es = getService('es');
+
+  describe('find_cases', () => {
+    afterEach(async () => {
+      await deleteAllCaseItems(es);
+    });
+
+    it('should return empty response', async () => {
+      const cases = await findCases({ supertest });
+      expect(cases).to.eql(findCasesResp);
+    });
+
+    it('should return cases', async () => {
```

Comment:
Could we add a test that tries to call find with `owner: 'securitySolution'` should return an error.